pkg/vcs/github_client/message.go

```diff
@@ -17,6 +17,63 @@ import (
 
 const MaxCommentLength = 64 * 1024
 
+const sepEnd = "\n```\n</details>" +
+"\n<br>\n\n**Warning**: Output length greater than max comment size. Continued in next comment."
+
+const sepStart = "Continued from previous comment.\n<details><summary>Show Output</summary>\n\n"
+
+// SplitComment splits comment into a slice of comments that are under maxSize.
+// It appends sepEnd to all comments that have a following comment.
+// It prepends sepStart to all comments that have a preceding comment.
+func SplitComment(comment string, maxSize int, sepEnd string, sepStart string) []string {
```

Comment:
can we make this unexported?

```suggestion
func splitComment(comment string, maxSize int, sepEnd string, sepStart string) []string {
```

Reply:
sure, done